crates/stages/src/stages/tx_lookup.rs

```diff
@@ -75,7 +76,7 @@ impl<DB: Database> Stage<DB> for TransactionLookupStage {
         }
 
         // Sort before inserting the reverse lookup for hash -> tx_id.
-        tx_list.sort_by(|txa, txb| txa.0.cmp(&txb.0));
+        tx_list.par_sort_unstable_by(|txa, txb| txa.0.cmp(&txb.0));
```

Comment:
how many do we have here?
always the commit threshold?
should we use this conditionally depending on size?

also, just noticed we can allocate the `tx_list` vec in like 67 with the exact capacity, right?

Reply:
> how many do we have here? 
> always the commit threshold?
Since we're doing it over block windows, the number of transactions is not possible to know. Maybe we can change so that we do the threshold check over the number of transactions instead. (in another PR)

>  should we use this conditionally depending on size?

Yeah, probably... but how would one decide on the best length before it uses `par`?

> also, just noticed we can allocate the tx_list vec in like 67 with the exact capacity, right?
yea!  :+1:  improved the benchmark further 6 %!